ark/mibi/tiling_utils.py

```diff
@@ -1150,15 +1102,36 @@ def remap_and_reorder_tiles(tiled_regions_proposed, proposed_to_auto_map,
     if moly_interval <= 0:
         raise ValueError("moly_interval must be at least 1")
 
-    # rename the FOVs based on the mapping
+    # define a new tiled regions proposed dict
+    tiled_regions_remapped = {}
+
+    # if the user specifies additional metadata, make sure they're copied over
+    # specifically, id, name, and status
+    if 'id' in tiled_regions_proposed:
+        tiled_regions_remapped['id'] = tiled_regions_proposed['id']
+    if 'name' in tiled_regions_proposed:
+        tiled_regions_remapped['name'] = tiled_regions_proposed['name']
+    if 'status' in tiled_regions_proposed:
+        tiled_regions_remapped['status'] = tiled_regions_proposed['status']
+
+    # define a new FOVs list for tiled_regions_remapped
+    tiled_regions_remapped['fovs'] = []
+
+    # rename the FOVs based on the mapping and append to tiled_regions_remapped
     for fov in tiled_regions_proposed['fovs']:
-        fov['name'] = proposed_to_auto_map[fov['name']]
+        # needed to prevent early saving since interactive visualization cannot stop this
+        # from running if a mapping_path provided already exists
+        fov_data = copy.deepcopy(fov)
+
+        # remap the name and append to tiled_regions_remapped
+        fov_data['name'] = proposed_to_auto_map[fov['name']]
+        tiled_regions_remapped['fovs'].append(fov_data)
 
     # randomize the order of the FOVs
-    tiled_regions_proposed['fovs'] = shuffle(tiled_regions_proposed['fovs'])
+    tiled_regions_remapped['fovs'] = shuffle(tiled_regions_remapped['fovs'])
```

Comment:
This should be controlled by a boolean flag for whether randomization happens or not